clap_man/CONTRIBUTING.md

```diff
@@ -0,0 +1,7 @@
+# How to Contribute
+
+See the [clap-wide CONTRIBUTING.md](../CONTRIBUTING.md).  This will contain `clap_generate` specific notes.
+
+### Scope
+
+`clap_man` contains the generator for creating manpages.
```

Comment:
This isn't needed.  In `clap_generate`, it was to help clarify when things should be in `clap_generate` vs another crate

Reply:
Done.